libs/cli/setup_test.go

```diff
@@ -61,12 +63,23 @@ func tempDir(t *testing.T) string {
 	return cdir
 }
 
+// writeConfigVals writes a toml file with the given values.
+// It returns an error if writing was impossible.
+func writeConfigVals(dir string, vals map[string]string) error {
+	data := ""
+	for k, v := range vals {
+		data += fmt.Sprintf("%s = \"%s\"\n", k, v)
+	}
```

Comment:
I guess this is pre-existing, but are we worried about escaping here?

```suggestion
	var lines []string
	for k, v := range vals {
		lines = append(lines, fmt.Sprintf("%s = %q", k, v)
	}
	data := strings.Join(lines, "\n")
```